src/hevm/src/EVM.hs

```diff
@@ -1246,6 +1245,12 @@ exec1 = do
         xxx ->
           vmError (UnrecognizedOpcode xxx)
 
+transfer :: Addr -> Addr -> Word -> EVM ()
+transfer xFrom xTo xValue =
+  zoom (env . contracts) $ do
```

Comment:
what does `zoom` do?

Reply:
it's a lens operator that allows you to zoom in on a substructure and modify it, without having to refer to its place in the larger structure all the time

Reply:
in other words it makes the `. env . contracts` implicits in all further lens operations